codice/ddf
# DDF-1617: renamed anonymous user to 'guest'

To test: ensure all i-tests pass, 

* start up, ensure anonymous configuration defaults to "guest"
* sign out of search ui, execute a query, check logs for guest user.


[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/codice/ddf/331)
